api/tests/test_handlers.py

```diff
@@ -0,0 +1,44 @@
+from django.test import TestCase
+
+from api.handlers import issue_handler
```

Comment:
https://github.com/FundersClub/FundersClub/wiki/Style-Guide:-Python

`api` is the same module import